webapp/src/components/root_portal.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import {Provider} from 'react-redux';
+
+import Conference from './conference/';
+
+type Props = {
+    children: React.ReactNode,
+}
+
+export default class RootPortal {
+    el: HTMLElement;
+    store: any;
+
+    constructor(registry: any, store: any) {
```

Comment:
Is there any chance we can remove the `any` from here?

Reply:
Probably we would need to have typescript types exposed somehow from the plugins code of mattermost, so I think we can have that in the future, but for now this is the simplest way.